centreon/src/Core/ServiceTemplate/Infrastructure/API/AddServiceTemplate/AddServiceTemplateOnPremSchema.json

```diff
@@ -0,0 +1,239 @@
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "notification_timeperiod_id": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "notification_type": {
+            "type": "integer"
+        },
+        "first_notification_delay": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "recovery_notification_delay": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "acknowledgement_timeout": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "freshness_checked": {
+            "type": "integer",
+            "enum": [
+                0,
+                1,
+                2
+            ]
+        },
+        "freshness_threshold": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "flap_detection_enabled": {
+            "type": "integer",
+            "enum": [
+                0,
+                1,
+                2
+            ]
+        },
+        "low_flap_threshold": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "high_flap_threshold": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "event_handler_enabled": {
+            "type": "integer",
+            "enum": [
+                0,
+                1,
+                2
+            ]
+        },
+        "event_handler_command_id": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "event_handler_command_args": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        },
+        "graph_template_id": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "note": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "note_url": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "action_url": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "icon_id": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "icon_alternative": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "severity_id": {
+             "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "is_activated": {
+            "type": "boolean"
+        },
+        "is_locked": {
+            "type": "boolean"
```

Comment:
Relates to comments on doc API, creation of service template with is_locked = true is not accepted outside of plugin/monitoring connectors installation (to confirm with @omercier if needed)
```suggestion
```